docs/apache-airflow-providers-amazon/index.rst

```diff
@@ -87,6 +87,7 @@ PIP package              Version required
 ``redshift_connector``   ``~=2.0.888``
 ``sqlalchemy_redshift``  ``~=0.8.6``
 ``watchtower``           ``~=2.0.1``
+``boto3_stubs[rds]``     ``>=1.20.46``
```

Comment:
i think this file should not be edited as it created automatically by the process of prepare provider release
cc @potiuk

Reply:
No worries. It wll get overridden automatically - I plan to start preparing providers tomorrow!